version/version.go

```diff
@@ -5,7 +5,11 @@ var (
 	GitCommit string
 
 	// Version is the built softwares version.
-	Version = TMCoreSemVer
+	Version string
+
+	// TMCoreSemVer is the current version of Tendermint Core.
+	// It's the Semantic Version of the software.
+	TMCoreSemVer = Version
```

Comment:
Do we have to introduce a new variable for this? Can we just use `TMCoreSemVer` directly?

Reply:
yes, within `.goreleaser.yml` there is a checksum section.

Reply:
That was my thought as well :)